app/views/homescreen/blocks/_administration.html.erb

```diff
@@ -0,0 +1,47 @@
+<h3 class="homescreen-blocks--item-header">
+  <span class="icon-context icon-settings"></span>
+  <%= l(:label_administration) %>
+</h3>
+
+<ul class="homescreen--arrow-links">
```

Comment:
I am asking myself whether those lists will occur in other places of the application. Then `-arrows` or `-arrow-items` might also be good class names...